libs/ui/src/remove_card_screen.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Screen } from './screen';
+import { Main } from './main';
+import { Prose } from './prose';
+import { fontSizeTheme } from './themes';
+
+const RemoveCardImage = styled.img`
+  margin: 0 auto -1rem;
+  height: 30vw;
+`;
+
+export function RemoveCardScreen(): JSX.Element {
+  return (
+    <Screen white>
+      <Main centerChild>
+        <Prose textCenter theme={fontSizeTheme.medium}>
+          <p>
+            <RemoveCardImage
+              aria-hidden
+              src="assets/remove-card.svg"
+              alt="Remove Card Icon"
+            />
+          </p>
```

Comment:
Since this `p` just contains an image and no text, let's remove the `p` wrapping (and add styles as needed to `RemoveCardImage`)